.circleci/config.yml

```diff
@@ -578,6 +578,12 @@ jobs:
       - run_tox_scenario:
           pattern: '^pynamodb_contrib-'
 
+  pyodbc:
```

Comment:
We also need to add entries below to actually have the job run in CI.

Reply:
I added these lines. These would be not needed after it will have been merged and datadog/dd-trace-py:latest have been updated.
I will create another pull request after merged.
https://github.com/DataDog/dd-trace-py/pull/1718/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R586-R588

Reply:
perfect! thanks 😄

Reply:
@kitsuyui we can do that follow-up PR now 🙂

Reply:
Sorry for my late.
Thanks for your follow up on #1729. 😄

Reply:
no problem at all 🙂